### PR TITLE
[DX-1861] Update node messaging

### PR DESCRIPTION
**Pull request by hamermike**

### CODE REVIEW

File: guides/sdks/node/usage-guide-messaging.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+```
+
+---
+
+## Send Text Message
+
+### 3.x
+
+```js
+const response = await controller.createMessage('user-id-123', {
```
Comment (ajrice6713) [inline guides/sdks/node/usage-guide-messaging.md]: Shouldn't this be `accountId`

File: guides/sdks/node/usage-guide-messaging.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+```js
+var body = new BandwidthMessaging.MessageRequest({
+    "applicationId" : "app-id-123",
+    "to"            : ["+19999999999"],
+    "from"          : "+18888888888",
+    "text"          : "The quick brown fox jumps over the lazy dog."
+});
+
+var response = await controller.createMessage("user-id-123", body);
```
Comment (ajrice6713) [inline guides/sdks/node/usage-guide-messaging.md]: Same here - `accountId`

**Review by jmulford-bw — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-update-node-messaging.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-update-node-messaging.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-update-node-messaging.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-update-node-messaging.s3-website-us-east-1.amazonaws.com/index.html

**Review by ajrice6713 — CHANGES_REQUESTED** [review]

File: guides/sdks/node/usage-guide-messaging.md (hunk 2 above)
Comment (hamermike) [reply guides/sdks/node/usage-guide-messaging.md]: Old file that I accidentally added, it wasn't being linked to. Good catch.

**Review by ajrice6713 — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-update-node-messaging.s3-website-us-east-1.amazonaws.com/index.html

**Pull request merged**